styles/work-section.css

```diff
@@ -81,6 +85,10 @@
   font-weight: bold;
 }
 
+.first-see-project-button-desktop {
+  display: none;
+}
+
 .project-1 {
```

Comment:
Well done for implementing the card having the see project button. As represented on Figma, the card having the project button represents how each of the cards should look when hovered while the rest cards without the button represent how each card should normally look.

Kindly implement this.

Figma design:
![image](https://user-images.githubusercontent.com/20554583/208044305-88feada4-2942-4d6c-9d26-ed0d79c0147f.png)